### ISSUE TITLE
`consistent-name` rule

### ISSUE THREAD

**Issue by jfmengels** [jfmengels/eslint-plugin-lodash-fp#2]
Enforce a consistent name for the imported Lodash, with `_` as the default.
Only applies to the default import, not of any cherrypicked method.

### Fail

```js
import lodash from 'lodash';
import foo from 'lodash/fp';
var bar = require('lodash');
var baz = require('lodash/fp');
```

### Pass

```js
import _ from 'lodash';
import _ from 'lodash/fp';
var _ = require('lodash');
var _ = require('lodash/fp');

/* eslint lodash-fp/consistent-name: ["error", "fp"] */
var fp = require('lodash/fp');
```
